assets/src/components/detours/detourMap.tsx

```diff
@@ -378,6 +393,47 @@ const OriginalRouteShape = ({
   </>
 )
 
+interface UnfinishedDiversionRouteShapeProps {
+  interactive: boolean
+  segments: UnfinishedRouteSegments
+  onClick: (e: LeafletMouseEvent) => void
+}
+
+const UnfinishedDiversionRouteShape = ({
+  interactive,
+  segments: { beforeStartPoint, afterStartPoint },
+  onClick,
+}: UnfinishedDiversionRouteShapeProps) => (
+  <>
+    <Polyline
+      weight={6}
+      interactive={false}
+      positions={beforeStartPoint.map(shapePointToLatLngLiteral)}
+      className="c-detour_map--original-route-shape-core"
+    />
+    <Polyline
+      weight={6}
+      interactive={false}
+      positions={afterStartPoint.map(shapePointToLatLngLiteral)}
+      className="c-detour_map--original-route-shape-core"
+    />
+    {interactive && (
+      <Polyline
+        positions={afterStartPoint.map(shapePointToLatLngLiteral)}
+        weight={16}
+        className={joinClasses([
+          "c-detour_map--original-route-shape-unfinished-diversion",
+          "c-detour_map--original-route-shape__unfinished",
```

Comment:
The combination of these two _totally_ different CSS naming conventions is gross, but I was thinking of waiting until a bit later to clean it up, because exactly _how_ to clean it up may depend on what happens when we remove the test group.

(It's possible that after doing some user-testing, Kathleen will decide that this wasn't the right way to signal the route-direction to dispatchers and we'll wind up backing out all of these changes. I think that's unlikely, but it is possible)

Reply:
Why is `c-detour_map--original-route-shape-unfinished-diversion` needed? It could be `c-detour_map--original-route-shape`, which seems to me a semantically accurate classname here.

Reply:
Ohhhhhhhhhh.... is it for the test??

Reply:
Yep. This is so that I can easily assert that the route shape that's displayed has changed.

Reply:
Although I guess I could assert on the number of `core` shapes there are...

This might affect https://github.com/mbta/skate/pull/2641 also.

Reply:
Oohhh actually, maybe [this would be a better way to make these assertions](https://github.com/mbta/skate/blob/e14d100c1db7df8bbf3c24ca4f8dbbe100d24d1e/assets/tests/components/detours/detourMap.test.tsx#L235-L273)?